tshirts.c

```diff
@@ -14,9 +14,23 @@ char size(int cms) {
 }
 
 int main() {
-    assert(size(37) == 'S');
-    assert(size(40) == 'M');
-    assert(size(43) == 'L');
+    //Standard T-shirt size based on chest measurement 
+    // S - 38, M - 40, L - 42, XL - 44, XXL - 46, 3XL - 48, 4XL - 50
+    assert(SizeOfTShirt(38) == 'S');

```

Comment:
Boundary values need to be asserted and also values before and after boundary values.
For ex if M= 40, then and S is <40, it is good to define this behavior with test cases having input 39, 40 and 41.